src/renderer/layouts/WorkspaceLayout.tsx

```
@@ -487,7 +487,8 @@ export default function WorkspaceLayout(): React.JSX.Element {
     maxTerminals,
     isWorkspaceRoute,
     cycleTab,
-    activeTab
+    activeTab,
+    handleCreateTerminalInPane
```

Biome (2.4.4)

[error] This variable is used before its declaration.

(lint/correctness/noInvalidUseBeforeDeclaration)